### ISSUE TITLE
Time zones!!

### ISSUE THREAD

**Issue by sfregosi** [PIFSC-Protected-Species-Division/cruise-maps-live#29]
The ship will spend time in HST and American Samoa Time when it is up in the NWHI. The Cruise Leader will inform us when they will be switching time zones, and it will always happen at night when surveying is not on going. All DAS files are recorded in 'local' time on the ship, but all acoustic files are in UTC and the GPX output files have a timezone reference that must be manually assigned.

Need to deal with this!

**Comment by ybarkley** [comment]
Thank you for catching this detail!
Did you know that we already switched to AST on 8/12?

**Comment by sfregosi** [comment]
@ybarkley yes! I saw it in Marie's updated on the 11th and so I clarified with her that it will always happen at night and that you all will be in AST/-11 until about the 23rd. 

My approach is:
- Created a 'time zone key' csv file where we will have to manually enter start/end dates for time in each zone for each leg. 
- Added a function called `assignTimeZone` that is called within `run.R` and takes a processed DAS, checks the date against the key, and assigns the timezone correctly. 
- All else runs on the `df_proc` processed DAS file output so will have correct timezone
- The compiled files now contain data collected in both timezones, but the single `DateTime` column does not like having two, so dates and times in there are all in HST. All this means is that if it was sighted at 17:08 AST, the sightings data.frame says it was at 18:08 HST. But each entry contains correct timezone info so all can be converted to/from UTC without issue. This does not affect the map or summary table. 

I reprocessed `DASALL.812` and `DASALL.813` so those data outputs are now correct and all complied data frames are correct.
